### ISSUE TITLE
Missing UIs

### ISSUE THREAD

**Issue by Darr3n2GG** [Darr3n2GG/Puzzle-Knights-2P-#1]
Missing UIs such as main menu,settings,next level text, game save etc

**Comment by Darr3n2GG** [comment]
main menu /

**Comment by Bryan-CoderS** [comment]
Setting? Like volume of sound? 
Game save?

**Comment by Darr3n2GG** [comment]
> Setting? Like volume of sound? 

Yup but it's optional( maybe also controls and gameplay changes like hard mode or smth)

> Game save?

Save and lod game system, also optional
